### DISCUSSION TITLE
getting an error ---> create "" failed (fetches failed) latest_version - failed converting

### DISCUSSION THREAD

**Discussion by prince20112003** [release-argus/Argus#360] [General]
I am getting this error when i try to add a software named greenbone to argus. 
the link to the webpage is : https://www.greenbone.net/en/documents/
RegEx: \d+\.\d+

[error-greenbone.txt](https://github.com/release-argus/Argus/files/14244675/error-greenbone.txt)

**Comment by JosephKav** [comment]
Sorry for missing this, but on the latest releases, that URL and RegEx give the version `1868`. That doesn't look right, but you can get GMP (22.4) or OSP 22.5) versions with `gmp-(\d+\.\d+)` or `osp-(\d+\.\d+)`
